perl-package/AI-MXNet/lib/AI/MXNet/Gluon/Loss.pm

```diff
@@ -75,19 +75,16 @@ method _apply_weighting(Str $F, GluonInput $loss, Maybe[Num] $weight=, Maybe[Glu
     return $loss;
 }
 
-# for symbolic output.shape is not available so we reshape
-# to empty shape and let it be inferred from output's shape
-# via the '-' operator later.
-
-method _reshape_label_as_output(GluonClass $F, GluonInput $output, GluonInput $label)
+# Reshapes x to the same shape as y
+method _reshape_like(GluonClass $F, GluonInput $x, GluonInput $y)
```

Comment:
this is an API change, right?

Reply:
No, subroutines that start with underscore are private functions and are not part of public API.